modules/sdk-unified-wallet/src/ecdsa-unified-wallet/ecdsaEVMUnifiedWallets.ts

```diff
@@ -4,12 +4,16 @@ import {
   KeychainsTriplet,
   SupplementGenerateWalletOptions,
   Keychains,
+  Wallet,
 } from '@bitgo/sdk-core';
 import { UnifiedWalletID, UnifiedWallet, GenerateUnifiedWalletOptions } from '../types';
-import { supportedCoins } from './types';
+import { supportedCoins, EvmWalletCoins } from './types';
 import { UnifiedWallets } from '../unifiedWallets';
+import _ from 'lodash';
```

Comment:
NIT: You could avoid using lodash as most of the functionalities are supported natively. Take a look at [this](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_isempty) for some suggestions